RapidWebView/src/main/AndroidManifest.xml

```diff
@@ -3,8 +3,9 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="in.workindia.rapidwebview">
 
-    <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
```

Comment:
This is an optional permission (part of JS Interface), please remove it from the package's declaration and add it in the documentation that the developer implementing this JS Interface has to include these permissions in their app manifest.